### PR TITLE
 Updated RepoTask module with new supported KeyValue pairs

### PR DESCRIPTION
**Pull request by shahabhijeet**
As TestFramework support custom env. updating the module to create proper connection string for the newly supported key value pairs.

### CODE REVIEW

File: tools/Repo-Tasks.psd1 [hunk 1]
```diff
@@ -30,10 +30,10 @@ CompanyName = 'Microsoft'
 Copyright = '(c) 2016 shahabhijeet. All rights reserved.'
 
 # Description of the functionality provided by this module
-# Description = 'Provides set of cmdlets for common repository tasks'
+# Description = ''
 
 # Minimum version of the Windows PowerShell engine required by this module
-# PowerShellVersion = '5.1'
+# PowerShellVersion = ''
```
Comment (markcowl) [inline tools/Repo-Tasks.psd1]: Don't we still need this?  I think we do.

File: tools/Repo-Tasks.psd1 [hunk 2]
```diff
@@ -42,13 +42,13 @@ Copyright = '(c) 2016 shahabhijeet. All rights reserved.'
...
 # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-# DotNetFrameworkVersion = '4.5'
+# DotNetFrameworkVersion = ''
 
 # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-# CLRVersion = '4.0'
+# CLRVersion = ''
 
 # Processor architecture (None, X86, Amd64) required by this module
-# ProcessorArchitecture = 'x86, Amd64'
```
Comment (markcowl) [inline tools/Repo-Tasks.psd1]: should be None

**Review by markcowl — CHANGES_REQUESTED** [review]
Please leave in the old constraints in the module manifest (minimum powershell version, .nte version, clr version,etc.

Otherwise, this looks good.

File: tools/Repo-Tasks.psd1 (hunk 2 above)
Comment (shahabhijeet) [inline tools/Repo-Tasks.psd1]: @markcowl no matter how much I try, Update-ModuleManifest will not add/update None value. I have now added that manually.

File: tools/Repo-Tasks.psd1 (hunk 1 above)
Comment (shahabhijeet) [inline tools/Repo-Tasks.psd1]: I believe Update-ModuleManifest made those changes. I will add it back.

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Pull request merged**